src/promptflow/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
   - Please set before importing `promptflow`, otherwise it won't take effect.
 - [Executor] Handle KeyboardInterrupt in flow test so that the final state is Canceled.
 - [Executor] Calculate system_metrics recursively in api_calls.
+- [Executor] Add root level api_calls.
```

Comment:
Let's make this line easier to understand for users who are reading the release note?

Reply:
Thanks for reminding. Updaged.